observation_portal/requestgroups/serializers.py

```diff
@@ -758,7 +760,7 @@ def create(self, validated_data):
             'tracking_num': request_group.id,
             'name': request_group.name
         }})
-        cache.set('observation_portal_last_change_time', timezone.now(), None)
+        cache.set('observation_portal_last_change_time_all', timezone.now(), None)
```

Comment:
This probably doesn't matter since it would more often than not be in the microseconds, but do you see any issues with the "all" last change time being slightly different from the most recent telescope_class last change time since `timezone.now()` is called for every call to `cache.set(...)`?

Reply:
i think microseconds don't matter (since we just use this to determine if we need to re-schedule yet or not), but I could change it to use the same now...

Reply:
Yeah.. the reason I lean towards changing it is mostly cause having the times be exactly the same is what the user would expect, so if there's a bit of a difference it might be momentarily confusing. The test `test_last_change_date_telescope_class_is_updated_when_request_is_submitted` even checks that those times are equal. Which pass of course because it uses mocked time, but it's still slightly different from the behavior of the running app. I'll leave it up to you.

Reply:
Okay, I've changed all of them to use the same now value for _all and per telescope caches. I agree this seems fine and means there will be less discrepancies